src/user-guide/writing-suites/configuration.rst

```diff
@@ -0,0 +1,211 @@
+.. _User Guide Configuration:
+
+Workflow Configuration
+======================
+
+Cylc suites are defined in structured, validated, :cylc:conf:`flow.cylc`
+files that concisely specify the properties of, and the relationships
+between, the various tasks managed by the suite.
+
+Here we will look at the format and content of the
+:cylc:conf:`flow.cylc` file and how to configure suites.
+
+
+.. _SuiteDefinitionDirectories:
+
+Suite Configuration Directories
+-------------------------------
+
+A Cylc :term:`suite directory` contains:
+
+:cylc:conf:`flow.cylc`
+   The file which configures the workflow.
+``bin/`` (optional)
+   A directory where you can put scripts and executables for use
+   in the workflow. It is automatically added to ``$PATH`` in the job
+   execution environment.
+
+   Alternatively, tasks can call external commands; or they can be
+   scripted entirely within the flow.cylc file.
+``lib/python/`` (optional)
+   A directory for Python modules which can be used for:
+
+   - Tasks; this directory is automatically added to ``$PYTHONPATH``
+     in the job execution environment.
+   - Custom :ref:`job submission modules <CustomJobSubmissionMethods>`.
+   - Custom :ref:`Jinja2 Filters<CustomJinja2Filters>`).
+
+Other files and folders may be placed in the :term:`suite directory` e.g.
+documentation, configuration files, etc.
+
+
+.. _FlowConfigFile:
+
+flow.cylc File Overview
+-----------------------
+
+:cylc:conf:`flow.cylc` files are an extended-INI format with section nesting.
```

Comment:
Hostage to fortune putting external links in, but is it worth putting in a link to somewhere on what an INI format is? It's pretty long in the tooth.